examples/sdk-interactive-examples/examples-template/src/example/Example.tsx

```diff
@@ -0,0 +1,11 @@
+// (C) 2021 GoodData Corporation
+import React from "react";
+
+/*
+import * as Catalog from "../catalog.js";
+import Hint from "../Hint.js";
+*/
+
+export default () => {
+    return <>this is a example template</>;
```

Comment:
Should we include `Hint` jsx by default, so people don't forget about it?

Reply:
Ok good point